.codeclimate.yml

```diff
@@ -1,3 +1,35 @@
+version: "2"
+checks:
+  argument-count:
+    config:
+      threshold: 4
+  complex-logic:
+    config:
+      threshold: 4
+  file-lines:
+    config:
+      threshold: 250
```

Comment:
This will be quite challenging for some *.test.js files at least.

Reply:
Well, this is a default value, so it was already like this for some time. I agree that for tests it's too small, but for production code I think this is a reasonable value. We could add tests into ignore list. I'll see what config options there are.